lib/beacon/router.ex

```diff
@@ -153,6 +156,19 @@ defmodule Beacon.Router do
     end
   end
 
+  @doc """
+  Creates a beacon sitemap index.
```

Comment:
I guess you already knot that but we'll need more docs here :)
Or a recipe to show how to use sitemaps.